Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -190,10 +198,37 @@ protected virtual void AttachToNewTrackedObject()
                 case TrackedObjectType.Head:
                     TrackTransform(CameraCache.Main.transform);
                     break;
-                // Other cases will come online as ControllerFinder is ported appropriately.
+                case TrackedObjectType.LeftController:
+                    var leftController = GetVisualizerForHand(Handedness.Left);
+                    if (leftController != null)
+                    {
+                        TrackTransform(leftController.gameObject.transform);
+                    }
+                    break;
+                case TrackedObjectType.RightController:
+                    var rightController = GetVisualizerForHand(Handedness.Right);
+                    if (rightController != null)
+                    {
+                        TrackTransform(rightController.gameObject.transform);
+                    }
+                    break;
+
+                    // Other cases will come online as ControllerFinder is ported appropriately.
             }
         }
 
+        private MixedRealityControllerVisualizer GetVisualizerForHand(Handedness hand)
```

Comment:
This isn't specific to SolverHandlers, and should be a utility function offered by another class/library. Otherwise we'll write it over and over again. Is there already a good place to move it?

Reply:
Agreed, I tried to make this is as generic as possible, but I take your point that "hand" probably isn't the correct verb here.